wrangler.toml

```diff
@@ -5,7 +5,7 @@ type = "webpack"
 webpack_config = "node_modules/flareact/webpack"
 
 [triggers]
-crons = ["* * * * *"]
+crons = ["*/2 * * * *"] # Workers KV free tier
```

Comment:
What about adding the KV free tier as a commented option?

Reply:
Oupsie, solved !